{"timestamp": "2024-02-19T06:37:52.043441+00:00", "level": "DEBUG", "message": "debug message", "module": "main", "line": 38, "function": "main", "thread": "MainThread", "logger": "my_app", "x": "hello"}
{"timestamp": "2024-02-19T06:37:52.044390+00:00", "level": "INFO", "message": "info message", "module": "main", "line": 39, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T06:37:52.045363+00:00", "level": "WARNING", "message": "warning message", "module": "main", "line": 40, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T06:37:52.045363+00:00", "level": "ERROR", "message": "error message", "module": "main", "line": 41, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T06:37:52.046343+00:00", "level": "CRITICAL", "message": "critical message", "module": "main", "line": 42, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T06:37:52.046343+00:00", "level": "ERROR", "message": "exception message", "module": "main", "line": 46, "function": "main", "thread": "MainThread", "logger": "my_app", "exc_info": "Traceback (most recent call last):\n  File \"d:\\repo\\repo\\autoDoc\\main.py\", line 44, in main\n    1 / 0\nZeroDivisionError: division by zero"}
{"timestamp": "2024-02-19T07:02:47.610440+00:00", "level": "DEBUG", "message": "debug message", "module": "main", "line": 38, "function": "main", "thread": "MainThread", "logger": "my_app", "x": "hello"}
{"timestamp": "2024-02-19T07:02:47.610440+00:00", "level": "INFO", "message": "info message", "module": "main", "line": 39, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T07:02:47.611416+00:00", "level": "WARNING", "message": "warning message", "module": "main", "line": 40, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T07:02:47.611416+00:00", "level": "ERROR", "message": "error message", "module": "main", "line": 41, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T07:02:47.612391+00:00", "level": "CRITICAL", "message": "critical message", "module": "main", "line": 42, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T07:02:47.612391+00:00", "level": "ERROR", "message": "exception message", "module": "main", "line": 46, "function": "main", "thread": "MainThread", "logger": "my_app", "exc_info": "Traceback (most recent call last):\n  File \"D:\\repo\\repo\\autoDoc\\main.py\", line 44, in main\n    1 / 0\n    ~~^~~\nZeroDivisionError: division by zero"}
{"timestamp": "2024-02-19T07:04:48.460213+00:00", "level": "DEBUG", "message": "debug message", "module": "main", "line": 38, "function": "main", "thread": "MainThread", "logger": "__main__", "x": "hello"}
{"timestamp": "2024-02-19T07:04:48.460213+00:00", "level": "INFO", "message": "info message", "module": "main", "line": 39, "function": "main", "thread": "MainThread", "logger": "__main__"}
{"timestamp": "2024-02-19T07:04:48.461188+00:00", "level": "WARNING", "message": "warning message", "module": "main", "line": 40, "function": "main", "thread": "MainThread", "logger": "__main__"}
{"timestamp": "2024-02-19T07:04:48.461188+00:00", "level": "ERROR", "message": "error message", "module": "main", "line": 41, "function": "main", "thread": "MainThread", "logger": "__main__"}
{"timestamp": "2024-02-19T07:04:48.462165+00:00", "level": "CRITICAL", "message": "critical message", "module": "main", "line": 42, "function": "main", "thread": "MainThread", "logger": "__main__"}
{"timestamp": "2024-02-19T07:04:48.462165+00:00", "level": "ERROR", "message": "exception message", "module": "main", "line": 46, "function": "main", "thread": "MainThread", "logger": "__main__", "exc_info": "Traceback (most recent call last):\n  File \"D:\\repo\\repo\\autoDoc\\main.py\", line 44, in main\n    1 / 0\n    ~~^~~\nZeroDivisionError: division by zero"}
{"timestamp": "2024-02-19T07:35:19.442076+00:00", "level": "DEBUG", "message": "debug message", "module": "log", "line": 31, "function": "main", "thread": "MainThread", "logger": "my_app", "x": "hello"}
{"timestamp": "2024-02-19T07:35:19.443052+00:00", "level": "INFO", "message": "info message", "module": "log", "line": 32, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T07:35:19.443052+00:00", "level": "WARNING", "message": "warning message", "module": "log", "line": 33, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T07:35:19.444029+00:00", "level": "ERROR", "message": "error message", "module": "log", "line": 34, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T07:35:19.444029+00:00", "level": "CRITICAL", "message": "critical message", "module": "log", "line": 35, "function": "main", "thread": "MainThread", "logger": "my_app"}
{"timestamp": "2024-02-19T07:35:19.445009+00:00", "level": "ERROR", "message": "exception message", "module": "log", "line": 39, "function": "main", "thread": "MainThread", "logger": "my_app", "exc_info": "Traceback (most recent call last):\n  File \"d:\\repo\\repo\\autoDoc\\example\\log.py\", line 37, in main\n    1 / 0\n    ~~^~~\nZeroDivisionError: division by zero"}
